test-app/src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/integration/IntegrationTestBase.kt

```diff
@@ -70,22 +70,20 @@ abstract class IntegrationTestBase {
   private val inboundTopic by lazy { hmppsQueueService.findByTopicId("inboundtopic") ?: throw MissingQueueException("HmppsTopic inboundtopic not found") }
 
   protected val inboundSqsClient by lazy { inboundQueue.sqsClient }
-  protected val inboundSqsDlqClient by lazy { inboundQueue.sqsDlqClient }
+  protected val inboundSqsDlqClient: AmazonSQS? by lazy { inboundQueue.sqsDlqClient }
```

Comment:
If we know this can never be null we could instead cast it to non-nullable using `as` (I think it would work anyway :smile: )
```
protected val inboundSqsDlqClient by lazy { inboundQueue.sqsDlqClient } as AmazonSQS
```

Reply:
I love that "AS"  - same for dlqurls etc